charts/nebraska/templates/secrets.yaml

```
@@ -1,6 +1,7 @@
-{{- $useGhAuth := or (eq .Values.config.auth.mode "github") (not .Values.config.auth.mode) }}
+{{- $useGhAuth := (eq .Values.config.auth.mode "github")}}
+{{- $useOidcAuth := (eq .Values.config.auth.mode "oidc")}}
 {{- $useDbPassword := not .Values.config.database.passwordExistingSecret.enabled }}
-{{- if or $useDbPassword $useGhAuth }}
+{{- if or (or $useDbPassword $useOidcAuth) ($useGhAuth) }}
```

```suggestion
{{- if or $useDbPassword $useOidcAuth $useGhAuth }}
```
The keyword `or` supports multiple arguments. Only examples shows 2 :-)

---

TIL